packages-exp/auth-exp/src/api/index.ts

```diff
@@ -65,28 +65,25 @@ export async function performApiRequest<T, V>(
   const errorMap = { ...SERVER_ERROR_MAP, ...customErrorMap };
   try {
     let body = {};
-    const params: { [key: string]: string } = {
-      key: auth.config.apiKey
-    };
+    let params = {};
     if (request) {
       if (method === HttpMethod.GET) {
-        Object.assign(params, request);
+        params = request;
       } else {
         body = {
           body: JSON.stringify(request)
         };
       }
     }
 
-    const queryString = Object.keys(params)
-      .map(key => {
-        return `${encodeURIComponent(key)}=${encodeURIComponent(params[key])}`;
-      })
-      .join('&');
-
     const response: Response = await Promise.race<Promise<Response>>([
       fetch(
-        `${auth.config.apiScheme}://${auth.config.apiHost}${path}?${queryString}`,
+        `${auth.config.apiScheme}://${auth.config.apiHost}${path}?${querystring(
```

Comment:
small nit: consider generating the query string first into a const then keeping this template string as a one-liner

Reply:
done